timelock-impl/src/main/java/com/palantir/atlasdb/timelock/AsyncTimelockServiceImpl.java

```diff
@@ -115,30 +110,53 @@ public RefreshLockResponseV2 refreshLockLeases(Set<LockToken> tokens) {
     }
 
     @Override
-    public StartAtlasDbTransactionResponse startAtlasDbTransaction(IdentifiedTimeLockRequest request) {
+    public StartAtlasDbTransactionResponse deprecatedStartTransaction(IdentifiedTimeLockRequest request) {
         return StartAtlasDbTransactionResponse.of(
                 lockImmutableTimestamp(request),
                 getFreshTimestamp());
     }
 
     @Override
-    public StartAtlasDbTransactionResponseV3 startIdentifiedAtlasDbTransaction(
+    public StartAtlasDbTransactionResponseV3 startTransaction(
             StartIdentifiedAtlasDbTransactionRequest request) {
+        StartTransactionResponseV4 startTransactionResponseV4 =
+                startTransactions(StartTransactionRequestV4.createForRequestor(request.requestorId(), 1));
+
+        return StartAtlasDbTransactionResponseV3.of(
+                startTransactionResponseV4.immutableTimestamp(),
+                getTimestampAndPartition(startTransactionResponseV4.timestamps()),
+                startTransactionResponseV4.lease()
+        );
```

Comment:
super nit: probably drop this on the same line for consistency with the rest of the codebase?